langchain/document_transformers/long_context_reorder.py

```diff
@@ -0,0 +1,44 @@
+"""Reorder documents"""
+from typing import Any, List, Sequence
+
+from pydantic import BaseModel
+
+from langchain.schema import BaseDocumentTransformer, Document
+
+
+def _litm_reordering(documents: List[Document]) -> List[Document]:
```

Comment:
Looks like this requires relevance scores to be pre-computed? Should we specify that?

Reply:
Yeap, this reordering only make sense 1 sec before inserting it into the final prompt.
No other value from it apart that.